index.php

```diff
@@ -70,10 +70,10 @@
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right" style="font-family: 'Montserrat', sans-serif;">
                     <li>
-                        <a class="page-scroll" href="aboutus/index.php">About</a>
+                        <a class="page-scroll" href="aboutus/index.php" style="color: rgb(224,255,255)">About</a>
```

Comment:
@prastut will correct it soon.

And on desktop the color anyway was the same. Isn't it?